iOS/SampleApp/SampleApp/ViewController.swift

```diff
@@ -12,31 +12,56 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-
 import UIKit
 import FVM
 
 class ViewController: UIViewController {
-    @IBOutlet weak var damageSelector: FVMCarModelViewController!
+    @IBOutlet weak var manageDescriptionTextView: UITextView!
+    @IBOutlet weak var manageDamagedCarPartsTextView: UITextView!
+    @IBOutlet weak var displayReturningJSON: UITextView!
+    @IBOutlet weak var goesToFVMButton: UIButton!
     
-    override func viewDidLoad(){
-        damageSelector.onStartup(jsonConfiguration: """
-        {
-            "mainScreenText": "text",
-            "selection":
-            [
-                {
-                    "id":"Roof"
-                },
-                {
-                    "id":"Hood"
-                }
-            ]
-        }
-        """)
-        
-        print(damageSelector.onCancel())
-        print(damageSelector.onAccept())
+    override func viewDidLoad() {
+        initializeManageDamagedCarPartsTextView()
+        initializeManageDescriptionTextView()
+        initializeDisplayReturningJSONTextView()
         super.viewDidLoad()
     }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        if CurrentValue.FirstTimeInformation == true {
+            showFirstTimeOpenAppInformationAlert()
+        }
+        else {
+            showInformationAlertWhenReturingFVM()
+        }
+        super.viewDidAppear(true)
```

Comment:
I think you should pass parameter here instead of const true

Reply:
It must be

Reply:
Why?

Reply:
Right